src/auto_pickup.cpp

```diff
@@ -357,7 +356,7 @@ void user_interface::show()
         }
         mvwprintz( w_header, point( 1, 3 ), c_white, "#" );
         mvwprintz( w_header, point( 8, 3 ), c_white, _( "Rules" ) );
-        mvwprintz( w_header, point( 52, 3 ), c_white, _( "I/E" ) );
+        mvwprintz( w_header, point( 52, 3 ), c_white, _( "Inc/Exc" ) );
```

Comment:
The column is this wide anyway to fit "Include" and "Exclude" in lower rows.

Reply:
Incl/Ex might be better
I read it as increment / execute (my dyslectic brain exc->exe).